CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Support of rocks repository specified in tt config.
 - ``cfg dump`` module. It prints tt environment configuration.
+- ``-use-docker`` option for ``tt pack``.
```

Comment:
Add a brief description about what this option does.

Reply:
Added.